### ISSUE TITLE
Windows 10 Speed

### ISSUE THREAD

**Issue by OllieB** [dcpurton/regfont#3]
We've noticed that regfont seems to operate much slower in Windows 10 than it did in Windows 7. Not sure why exactly, but the difference is huge when registering 1000+ fonts.

We have a script that registers a folder of fonts on our network, we're just not sure why it's slower on our Windows 10 machines than our Windows 7 ones

Any ideas what we might be able to do to improve performance?

Thanks!

**Comment by dcpurton** [comment]
Sorry, I'm not going to be able to be of much help. I haven't owned or used Windows for years.

I guess it's possible that Windows 10 does a lot more sanity checking of fonts than previous versions. Not a problem if you only register once, but more so if you're doing it every login.

Some things you could try:
- rebuild it using a Microsoft compiler (the Git version supports this, or at least did a while ago)
- is there a difference between the 64bit version compared to the 32bit version?
- run it in debug mode and try and pick up where the slow step is (Although on one font it might be hard to tell. I could maybe add time stamp prints to the debug output which might give a better idea.)

**Comment by OllieB** [comment]
Fair enough! I'll have a look at compiling it myself when I find some time, see if I can track down the issue.

Thanks!
